src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -162,10 +182,10 @@ public void testEditQuestionAction() throws Exception {
         ______TS("MSQ: edit question success");
 
         feedbackEditPage.clickEditQuestionButton(1);
-        feedbackEditPage.fillQuestionTextBox("edited msq qn text", 1);
-        feedbackEditPage.fillQuestionDescription("more details", 1);
+        feedbackEditPage.fillQuestionTextBox("edited msq qn text", QUESTION_INDEX_ONE);
```

Comment:
There is no need to change other places. We understand that the numeric number like `1`, `2` and `3` are questions' indexes. My previous review comment mainly focuses on the magic `-1`. It is not so obvious that the `-1` is also a question index.